service/iot-service-samples/pnp-service-sample/temperature-controller-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/service/TemperatureController.java

```diff
@@ -89,7 +95,7 @@ private static void InvokeMethodOnRootLevel() throws IOException, IotHubExceptio
     }
 
     private static void InvokeMethodOnComponent() throws IOException, IotHubException {
-        String methodToInvoke = PnpHelper.CreateComponentCommandName("thermostat1", "getMaxMinReport");
+        String methodToInvoke = "thermostat1" + "*" +  "getMaxMinReport";
```

Comment:
Can you add the comments we had for this as well in the helper earlier?

Reply:
coming in next commit